lib/utils/getLocationOptions.js

```diff
@@ -1,4 +1,4 @@
-export const getLocationOptions = (locations) => locations.map(({ name, id, code }) => ({
+export const getLocationOptions = (locations = []) => locations.map(({ name, id, code }) => ({
```

Comment:
`locations` could be passed as null and it will cause exception. just use `locations?.map ?? []`